examples/status-message/package.json

```diff
@@ -5,15 +5,18 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "build": "yarn build:simple && yarn build:collections",
+    "build": "yarn build:simple && yarn build:collections && yarn build:borsh",
     "build:simple": "near-sdk build src/status-message.js build/status-message.base64",
     "build:collections": "near-sdk build src/status-message-collections.js build/status-message-collections.base64",
+    "build:borsh": "near-sdk build src/status-message-borsh.js build/status-message-borsh.base64",
     "test": "ava"
   },
   "author": "Near Inc <hello@nearprotocol.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "near-sdk-js": "file:../../"
+    "borsh": "https://github.com/near/borsh-js/tarball/import-buffer",
```

Comment:
I do not understand how this branch is different and why are we using it here. Can we use `fix-build` branch instead? (or maybe master if we will merge it).

Reply:
It has a `import {Buffer} from `buffer/` in borsh.ts, and borsh.js built from that. Without this line it doesn't work.